app/renderer/suite.js

```diff
@@ -58,15 +58,15 @@ class Suite {
     getValidName(name) {
         let index = 2;
         if (!this.existTaskName(name)) return name;
-        while (this.existTaskName(`${name  } (${  index  })`)) {
+        while (this.existTaskName(`${name} (${index})`)) {
```

Comment:
`${name} (${index})` should be assigned to a constant with a name

Reply:
At the end of every iteration, we're increasing value of `index` by one, so if we make `${name} (${index})` part of a const, we'd have to change structure of this look such that assigned variable is also updated at the end of loop. I can do the change, but it would be slightly more for refactoring :thinking: what do you suggest?

Reply:
Well, not necessarily a const but a let.

mmm, you got a point there a do...while could solve this, but indeed, probably the result is worse than the problem. Lets leave it this way then